dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -761,39 +761,39 @@ else if(cont.getStructure().getStructureType()==Structure.STRUCTURE_TYPE_HTMLPAG
     @SuppressWarnings("unchecked")
 	private List<Map> getFoldersTree (Folder parent, Role[] roles) throws DotStateException, DotDataException, DotSecurityException {
         FolderAPI folderAPI = APILocator.getFolderAPI();
-        List<Folder> children = new ArrayList<Folder>();
+        List<Folder> children = new ArrayList<>();
 		try {
 			children = folderAPI.findSubFolders(parent,userAPI.getSystemUser(),false);
 		} catch (Exception e) {
 			Logger.error(this, "Could not load folders : ",e);
 		}
-        return getFoldersTree(parent.getInode(), children, roles);
+        return getFoldersTree(parent.getIdentifier(), children, roles);
     }
 
-	private List<Map> getFoldersTree (String parentInode, List<Folder> children, Role[] roles) throws DotStateException, DotDataException, DotSecurityException {
+	private List<Map> getFoldersTree (String parentId, List<Folder> children, Role[] roles) throws DotStateException, DotDataException, DotSecurityException {
 
         WebContext ctx = WebContextFactory.get();
         User usr = getUser(ctx.getHttpServletRequest());
         ArrayList<Map> folders = new ArrayList<Map> ();
 
-        for (Folder f : children) {
-        	Map<String, Object> folderMap = f.getMap();
-        	if (openFolders.contains(f.getInode())) {
-        		List<Map> childrenMaps = getFoldersTree (f, roles);
+        for (Folder folder : children) {
+        	Map<String, Object> folderMap = folder.getMap();
+        	if (openFolders.contains(folder.getIdentifier())) {
+        		List<Map> childrenMaps = getFoldersTree (folder, roles);
         		folderMap.put("open", true);
         		folderMap.put("childrenFolders", childrenMaps);
         	} else {
         		folderMap.put("open", false);
         	}
-        	if(f.getInode().equalsIgnoreCase(activeFolderInode))
+        	if(folder.getIdentifier().equalsIgnoreCase(activeFolderInode))
         		folderMap.put("selected", true);
         	else
         		folderMap.put("selected", false);
-        	folderMap.put("parent", parentInode);
+        	folderMap.put("parent", parentId);
 
         	List permissions = new ArrayList();
         	try {
-        		permissions = permissionAPI.getPermissionIdsFromRoles(f, roles, usr);
+        		permissions = permissionAPI.getPermissionIdsFromRoles(folder, roles, usr);
         	} catch (DotDataException e) {
         		Logger.error(this, "Could not load permissions : ",e);
```

Comment:
Let's log the ID of the parent folder here too.

Reply:
Done